oneandone/resource_oneandone_server.go

```diff
@@ -487,20 +487,8 @@ func resourceOneandOneServerUpdate(d *schema.ResourceData, meta interface{}) err
 			return err
 		}
 
-		o, n := d.GetChange("firewall_policy_id")
-		if n == nil {
-			for _, ip := range server.Ips {
-				mp, err := config.API.DeleteFirewallPolicyServerIp(o.(string), ip.Id)
-				if err != nil {
-					return err
-				}
-
-				err = config.API.WaitForState(mp, "ACTIVE", 30, config.Retries)
-				if err != nil {
-					return err
-				}
-			}
-		} else {
+		_, n := d.GetChange("firewall_policy_id")
```

Comment:
Can we update firewall policies on the server?

Reply:
At the moment, the API is returning HTTP 500 when I try to update the firewall policy.
Although, this was not in the scope of this task, I have made this specific change because the API does not support the DELETE method for the said resource anymore. This was causing an error and the tests were failing. I am assuming that the API works in such way that the POST will update the resource, since it is no longer possible to call the DELETE method.